packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -49,10 +52,31 @@ class InteractionLayerContainer extends Component {
       move,
       setActiveMark,
       scale,
+      transcribedLines,
       width
     } = this.props
     return (
       <>
+        <g id="transcribedLines">
+          {transcribedLines.map(line => {
+            const color = line.consensusReached ? 'green' : 'red'
```

Comment:
@wgranger please correct if I have a misunderstanding, but a transcription that has reached consensus is retired, yes? If so, the conditional here should be rendering the mark as grey here if consensus is reached and red if not.

Green indicates transcription marks in progress by the volunteer.

From the ASM tutorial:

<img width="370" alt="Screen Shot 2020-02-21 at 4 03 31 PM" src="https://user-images.githubusercontent.com/5016731/75075604-d2c98700-54c3-11ea-94e1-6a2be339fac4.png">

Reply:
Noting @beckyrother has final say here on the colors for the different states of the transcription mark lines.